llvm/lib/Target/AMDGPU/AMDGPUArgumentUsageInfo.h

```diff
@@ -143,6 +143,11 @@ struct AMDGPUFunctionArgInfo {
   ArgDescriptor WorkGroupInfo;
   ArgDescriptor PrivateSegmentWaveByteOffset;
 
+  // System TTMPs.
+  ArgDescriptor ArchitectedWorkGroupIDX;
```

Comment:
I don't think these really need to be tracked in ArgumentUsageInfo; they aren't arguments anymore

Reply:
OK. I started again from scratch: #79120